backend/authentication/models.py

```diff
@@ -49,21 +61,23 @@ def __str__(self) -> str:
 class User(AbstractUser, CreationDeletionMixin):
     id = models.UUIDField(primary_key=True, default=uuid4, editable=False)
     user_name = models.CharField(max_length=255)
-    name = models.CharField(max_length=255, blank=True)
+    name = models.CharField(max_length=255, blank=True, null=True)
     password = models.CharField(max_length=255)
     description = models.TextField(max_length=500)
     verified = models.BooleanField(default=False)
-    verification_method = models.CharField(max_length=30, blank=True)
+    verification_method = models.CharField(max_length=30, blank=True, null=True)
     verification_partner = models.ForeignKey(
         "User", on_delete=models.SET_NULL, null=True
     )
     user_icon = models.ForeignKey("content.Image", on_delete=models.SET_NULL, null=True)
     social_accounts = ArrayField(
         models.CharField(max_length=255), null=True, blank=True
     )
-    total_flags = models.IntegerField(default=0)
     private = models.BooleanField(default=False)
     high_risk = models.BooleanField(default=False)
+    creation_date = models.DateTimeField(auto_now_add=True)
```

Comment:
There are few minor things here: 
* Creation and deletion date are added by the CreationDeletionMixin. 
* And we got rid of the total_flags.

Reply:
We're going to have a central `flags` table now, @thiagogquinto :) So with that we don't need `total_flags`, but rather will be referencing the `flags` table via uuids.

Reply:
I removed the fields total_flags, but as there is an issue for the addition of the flag table (#626), I will not add on this PR, unless that's okay to add it.